src/components/Forms/TradieSignUp.js

```diff
@@ -0,0 +1,295 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Link from '@material-ui/core/Link';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import useForm from './useForm';
+import validate from './validateInfo';
+import Logo from "./Logo";
+
+
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '100vh',
+  },
+  paper: {
+    margin: theme.spacing(8, 4),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  buttonContainer:{
+      display: 'flex',
+      flexDirection: 'row-reverse',
+      paddingTop: '15vh',
+  },
+
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(10),
+  },
+  submit: {
+    margin: theme.spacing(3, 2, 3, 6),
+  },
+}));
+
+export default function TradieSignUp() {
+    const {handleChange, values, handleSubmit, errors} = useForm(validate);
+
+  const classes = useStyles();
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+       <Logo />
+      <Grid item xs={12} sm={7} md={7} component={Paper} elevation={6} square>
+        <div className={classes.paper}>
+          <Typography component="h1" variant="h4">
+            Tradies Sign up
+          </Typography>
+          <form className={classes.form}  onSubmit={handleSubmit}>
+          <Grid container spacing={3}>
+          <Grid container>
+              {/* <Grid item xs>
+                <Link href="#" variant="body2">
+                  Forgot password?
+                </Link>
+              </Grid> */}
```

Comment:
pls delete uncessary comments

Reply:
Sure.
I will clean up unenessary comments and parts of post request.